### PR TITLE
Typesafe i18n

### PR DESCRIPTION
**Pull request by notmd**
- Setup Typescript support for i18n
- Add missing translation

### CODE REVIEW

File: website/src/components/Header/UserMenu.tsx [hunk 1]
```diff
@@ -20,7 +20,6 @@ import { FiAlertTriangle, FiLayout, FiLogOut, FiSettings, FiShield } from "react
 interface MenuOption {
   name: string;
   href: string;
-  desc: string;
```
Comment (notmd) [inline website/src/components/Header/UserMenu.tsx:23]: I delete it because it's being used anywhere

File: website/src/components/Faq.tsx [hunk 2]
```diff
@@ -27,10 +27,10 @@ export function Faq() {
             return (
               <ListItem className="space-y-10" key={`question_${index}`}>
                 <Text as="h3" className={`text-lg font-semibold leading-6 ${headingColorClass}`}>
-                  {t(`faq_items.q${index}`)}
+                  {t(`faq_items.q${index as 0}`)}
                 </Text>
                 <Text as="p" className={`mt-4 text-sm ${textColorClass}`}>
-                  {t(`faq_items.a${index}`)}
+                  {t(`faq_items.a${index as 0}`)}
```
Comment (AbdBarho) [inline website/src/components/Faq.tsx:33]: oof! do we need to cast to a literal?

**Review by fozziethebeat — APPROVED** [review]

File: website/src/components/Faq.tsx (hunk 2 above)
Comment (notmd) [reply website/src/components/Faq.tsx:33]: Yes, otherwise TS will complain

**Comment by fozziethebeat** [comment]
Will merge once the conflict has been resolved

File: website/src/pages/dashboard.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { Flex } from "@chakra-ui/react";
 import Head from "next/head";
+import { useTranslation } from "next-i18next";
```
Comment (notmd) [inline website/src/pages/dashboard.tsx:3]: I fixed the wrong import introduced in #890

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**